client/my-sites/site/index.jsx

```diff
@@ -73,33 +135,52 @@ module.exports = React.createClass( {
 			'is-primary': site.primary,
 			'is-private': site.is_private,
 			'is-redirect': site.options && site.options.is_redirect,
-			'is-selected': this.props.isSelected
+			'is-selected': this.props.isSelected,
+			'is-toggled': this.state.showMoreActions
 		} );
 
 		return (
 			<div className={ siteClass }>
-				<a className="site__content"
-					href={ this.props.homeLink ? site.URL : this.props.href }
-					target={ this.props.externalLink && '_blank' }
-					title={ this.props.homeLink ? this.translate( 'Visit "%(title)s"', { args: { title: site.title } } ) : site.title }
-					onTouchTap={ this.onSelect }
-					onClick={ this.props.onClick }
-					onMouseEnter={ this.props.onMouseEnter }
-					onMouseLeave={ this.props.onMouseLeave }
-					aria-label={ this.translate( 'Open site %(domain)s in new tab', { args: { domain: site.domain } } ) }
-				>
-					<SiteIcon site={ site } />
-					<div className="site__info">
-						<div className="site__title">{ site.title }</div>
-						<div className="site__domain">{ site.domain }</div>
+				{ ! this.state.showMoreActions ?
```

Comment:
How would you split it? The more actions area should be a sub-component down the road.